### PR TITLE
Update FerretDB to version 1.0

### PR DESCRIPTION
**Pull request by rumyantseva**
- This PR provides a configuration for FerretDB 1.0. 
- The pod with the app consists of two containers: FerretDB itself and a separate PostgreSQL container. This approach follows the [official FerretDB documentation](https://docs.ferretdb.io/quickstart-guide/docker/).
- Only FerretDB's port (`27017`) is exposed as service port.

---

Existing Marketplace application checklist:

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [x] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [x] Tested that the application works with the proposed updates applied (including a screenshot).
* [x] Updated the version number of the app if that has changed.

### CODE REVIEW

File: ferretdb/app.yaml [hunk 1]
```diff
@@ -26,16 +37,41 @@ spec:
...
+              name: ferretdb
+        - image: postgres:15.2
+          name: postgresql
+          env:
             - name: POSTGRES_USER
               value: $POSTGRES_USER
             - name: POSTGRES_PASSWORD
               value: $POSTGRES_PASSWORD
+            - name: POSTGRES_DB
+              value: ferretdb
```
Comment (rumyantseva) [inline ferretdb/app.yaml:64]: @AlekSi please confirm that this approach is enough, and we don't need to create additional databases.

**Comment by rumyantseva** [comment]
@AlekSi @w84thesun @noisersup @chilagrow I tested this configuration locally on minikube, and it worked. Please take a look!

Next step: I'm going to test it on the civo infrastructure according to its documentation.

**Comment by saiyam1814** [comment]
> @AlekSi @w84thesun @noisersup @chilagrow I tested this configuration locally on minikube, and it worked. Please take a look!
> 
> Next step: I'm going to test it on the civo infrastructure according to its documentation.

This app is not yet available in production, it should be available by today.

**Comment by rumyantseva** [comment]
For now, I tested the "static" version of `app.yaml`:
- I created a test cluster through `civo k8s create`
- I replaced variables in `app.yaml` file with static values
- I run `kubectl apply -f app.yaml`

As a result, the service was deployed successfully.

Here is my screenshot:
Screenshot 2023-04-05 at 10 06 45

File: ferretdb/app.yaml (hunk 1 above)
Comment (AlekSi) [reply ferretdb/app.yaml:64]: It is

**Comment by AlekSi** [comment]
> This app is not yet available in production, it should be available by today.

@saiyam1814 Please review that PR, but don't make FerretDB available in the marketplace yet

**Comment by AlekSi** [comment]
This PR is ready to be merged. We extensively tested it.

**Pull request merged**